### DISCUSSION TITLE
PUBG: NEW STATE

### DISCUSSION THREAD

**Discussion by DreamzStudio** [gildor2/UEViewer#248] [Q&A]
While trying to view the .uasset files of PUBG: NEW STATE i got this error

Package "/Game/Art/00_Baltic/03_Building/Building/meshs/HouseSets/SM_Baltic_HS_Campsite_02_1.uasset": wrong export index 1126807819
UnPackage::CreateExport: /Game/Art/00_Baltic/03_Building/Building/meshs/HouseSets/SM_Baltic_HS_Campsite_02_1.uasset:1126807819 <- UnPackage::SerializeUObject <- TArray::Serialize: 0/1130626031 <- UStaticMesh4::Serialize <- LoadObject: StaticMesh4'SM_Baltic_HS_Campsite_02_1.SM_Baltic_HS_Campsite_02_1', pos=11136, ver=522/0 (unversioned), game=ue4.26 <- UObject::EndLoad <- LoadWholePackage: /Game/Art/00_Baltic/03_Building/Building/meshs/HouseSets/SM_Baltic_HS_Campsite_02_1.uasset <- CUmodelApp::ShowPackageUI <- Main: umodel_build=1576

plz help

**Comment by Blenux** [comment]
https://www.gildor.org/smf/index.php/topic,6073.30.html
